bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/console/HueCommandExtension.java

```diff
@@ -126,12 +235,14 @@ public boolean complete(String[] args, int cursorArgumentIndex, int cursorPositi
         if (cursorArgumentIndex <= 0) {
             return new StringsCompleter(thingRegistry.getAll().stream()
                     .filter(t -> HueBindingConstants.THING_TYPE_BRIDGE.equals(t.getThingTypeUID())
+                            || HueBindingConstants.THING_TYPE_BRIDGE_API2.equals(t.getThingTypeUID())
                             || HueBindingConstants.THING_TYPE_GROUP.equals(t.getThingTypeUID()))
                     .map(t -> t.getUID().getAsString()).collect(Collectors.toList()), true)
                     .complete(args, cursorArgumentIndex, cursorPosition, candidates);
         } else if (cursorArgumentIndex == 1) {
             Thing thing = getThing(args[0]);
-            if (thing != null && HueBindingConstants.THING_TYPE_BRIDGE.equals(thing.getThingTypeUID())) {
+            if (thing != null && (HueBindingConstants.THING_TYPE_BRIDGE.equals(thing.getThingTypeUID())
+                    || HueBindingConstants.THING_TYPE_BRIDGE_API2.equals(thing.getThingTypeUID()))) {
                 return SUBCMD_COMPLETER.complete(args, cursorArgumentIndex, cursorPosition, candidates);
```

Comment:
The commands are not the same for the two kind of bridges. So please define two different SUBCMD_COMPLETER, one for each bridge type.

Reply:
Ok. Done.